tests/utils/verification/volumeproperties.go

```diff
@@ -24,22 +24,22 @@ import (
 	sshutil "github.com/vmware/docker-volume-vsphere/tests/utils/dockercli"
 )
 
-var ADMIN_CLI_LS = "/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py volume ls "
-var DOCKER_CLI_INSPC = "docker volume inspect "
+var adminCliLs = "/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py volume ls "
```

Comment:
This is out of the scope of this PR. The idea behind this PR was to rename files and addressing golint warnings. Please share your idea if you still want to include as part of this PR or not.